scripts/2_munge/sankey_fig.R

```diff
@@ -6,12 +6,56 @@ arrows.1 = group_by(fish.change, X1989.2014) %>%
   summarize(toCoexistence=sum(X2040.2064 == 'Coexistence'),toWally=sum(X2040.2064 == 'Walleye dominant'), 
             toBass = sum(X2040.2064 == 'Bass dominant'), toNeither=sum(X2040.2064 == 'Neither'))
 
-
+min.h <- 3 #px
```

Comment:
I don't actually think there will be any conflicts, except that I deleted this entire file. These changes you made to the svg code will have to go into `visualizeFutureSuitability.R`. Now I am thinking we should merge mine first and then add the changes to this file into the new file ? I don't know what would be easiest